packages/features/ee/teams/components/TeamsListing.tsx

```diff
@@ -67,23 +34,29 @@ export function TeamsListing() {
 
       {isLoading && <SkeletonLoaderTeamList />}
 
-      <UpgradeScreen
-        title="calcom_is_better_with_team"
-        description="add_your_team_members"
-        features={features}
-        background="/team-banner-background.jpg"
```

Comment:
@JeroenReumkens  why did you remove this prop and hardcode `background: "url(/team-banner-background.jpg)` again?

the background has to be different for each UpgradeScreen